packages/teams-js/src/public/search.ts

```diff
@@ -127,4 +145,74 @@ export namespace search {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.search ? true : false;
   }
+
+  /**
+   * Clear the host M365 application's search box
+   *
+   * @beta
+   */
+  export function closeSearch(): Promise<void> {
+    return new Promise<void>((resolve) => {
+      ensureInitialized(runtime, FrameContexts.content);
+      if (!isSupported()) {
+        throw new Error('Not supported');
+      }
+
+      resolve(sendAndHandleError('search.closeSearch', version));
```

Comment:
Why do you need to pass the package version up to the host in order to close search?

Reply:
That was actually one of the questions we had for you if it was necessary, if you also see it as something not needed, I will delete it too